dask_imread/__init__.py

```diff
@@ -1,8 +1,108 @@
 # -*- coding: utf-8 -*-
 
+
 __author__ = """John Kirkham"""
 __email__ = "kirkhamj@janelia.hhmi.org"
 
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+
+import itertools
+import numbers
+import warnings
+
+import dask
+import dask.array
+import dask.delayed
+import numpy
+import pims
+
+tifffile = None
+try:
+    import tifffile
+except ImportError:
+    try:
+        from skimage.external import tifffile
+    except ImportError:  # pragma: no cover
+        pass
+
+
+# Monkey patch pims to use the available tifffile.
+if pims.tiff_stack.tifffile is None:
+    pims.tiff_stack.tifffile = tifffile
+
+
+def imread(fname, nframes=1):
```

Comment:
Instead of specifying the number of frames here it might make more sense to allow this to be chunks. We could handle this in a couple of ways.

1. Require the non-frame dimensions to be one chunk. (so either `None` or the full length)
2. Allow chunking along other dimensions, but only apply it with `rechunk` afterwards.